### PR TITLE
AT_04.25.01 | Create Booking page > Passengers details functionality - One passenger > Verify the opportunity to fill main passenger's name in 'Passenger name' input field

### PR DESCRIPTION
**Pull request by YurySelivanov**
https://trello.com/c/Sip7qADZ/259-at042501-create-booking-page-passengers-details-functionality-one-passenger-verify-the-opportunity-to-fill-main-passengers-name

### CODE REVIEW

**Comment by SvetlanaMag** [comment]
На митинге решили, что в fixtures data будем разбивать не по секциям страницы, а по элементам, т.е. файл должен называться не passengersDetails.json, а inputField.json

File: cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/US_04.25_functionality-One passenger.cy.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import {PassengersDetailsSection} from "../../../../pageObjects/CreateBookingPage.js";
+import {DepartureDateSection} from "../../../../pageObjects/CreateBookingPage.js";
+import {DepartureOnSection} from "../../../../pageObjects/CreateBookingPage.js";
+
+const passengersDetailsSection = new PassengersDetailsSection();
+const departureDateSection = new DepartureDateSection();
+const departureOnSection = new DepartureOnSection();
+
+describe('US_04.25 | functionality - One passenger', () => {
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/US_04.25_functionality-One passenger.cy.js:11]: US name doesn’t look right

Comment (YurySelivanov) [reply cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/US_04.25_functionality-One passenger.cy.js]: I see, it's my mistake, i've already copied the US-name from the Trello card

**Review by elviraKh1 — APPROVED** [review]

File: cypress/pageObjects/CreateBookingPage.js [hunk 2]
```diff
@@ -1,6 +1,32 @@
-class CreateBookingPage {
```
Comment (Rom-T) [inline cypress/pageObjects/CreateBookingPage.js:1]: Why delete this class and create other classes in sections in it? This is on different pages and not Popup windows, but just page blocks.

Comment (elviraKh1) [reply cypress/pageObjects/CreateBookingPage.js:1]: I am very sorry for my approval, I didn't check it properly

**Comment by YurySelivanov** [comment]
Thanks for your advices.  I've already almost edited all these mistakes

**Pull request merged**